ros2bag/ros2bag/verb/record.py

```diff
@@ -34,168 +34,177 @@
 import yaml
 
 
+def add_recorder_arguments(parser: ArgumentParser) -> None:
+    parser.formatter_class = SplitLineFormatter
+    writer_choices = get_registered_writers()
+    default_storage_id = get_default_storage_id()
+    default_writer = default_storage_id if default_storage_id in writer_choices else \
+        next(iter(writer_choices))
+
+    serialization_choices = get_registered_serializers()
+    converter_suffix = '_converter'
+    serialization_choices = {
+        f[:-len(converter_suffix)]
+        for f in serialization_choices
+        if f.endswith(converter_suffix)
+    }
+
+    # Base output
+    parser.add_argument(
+        '-o', '--output',
+        help='Destination of the bagfile to create, '
+             'defaults to a timestamped folder in the current directory.')
+    parser.add_argument(
+        '-s', '--storage', default=default_writer, choices=writer_choices,
+        help="Storage identifier to be used, defaults to '%(default)s'.")
+
+    # Topic filter arguments
+    topics_args_group = parser.add_mutually_exclusive_group()
+    topics_args_group.add_argument(
+        'topics_positional', type=str, default=[], metavar='[Topic ...]', nargs='*',
+        help='Space-delimited list of topics to record.')
+    topics_args_group.add_argument(
+        '--topics', type=str, default=[], metavar='Topic', nargs='+',
+        help='Space-delimited list of topics to record.')
+    parser.add_argument(
+        '--topic-types', nargs='+', default=[], help='List of topic types to record.')
```

Comment:
In order to maintain a consistent style of description, how about this type of option should use "space-delimited" as the prompt.
`List of ...` -> `Space-delimited list of`

Reply:
@Barry-Xu-2018 It's a bit unrelated to the matter of this PR, and I was planning to fix it in the follow-up PR. 
But since you found it and asking for fix, I will fix it in the scope of this PR.
Fixed in the new commit https://github.com/ros2/rosbag2/pull/1632/commits/875e7876b45c676b3ebcd14a69abf402e94aa796